docs/docs/services/qix-engine/access-control.md

```diff
@@ -4,21 +4,40 @@
     This feature is in an experimental state. Use with caution
     since this feature may change in the future.
 
-Being able to control which users can do what in the context
-of a document is often necessary. In Qlik Associative Engine we support
-[Attribute-Based Access Control (ABAC)](https://en.wikipedia.org/wiki/Attribute-based_access_control)
-which lets you control the document resources based on a users'
-attributes rather than using roles or other, more static, means.
+Being able to control which users can do what in the context of a document is often necessary. In Qlik Associative
+Engine we support [Attribute-Based Access Control (ABAC)](https://en.wikipedia.org/wiki/Attribute-based_access_control)
+which lets you control the document resources based on a user's attributes rather than using roles or other more static
```

Comment:
Just curious, why `user's` and not `users'`? https://english.stackexchange.com/a/36659

Reply:
It states "a user" - singular form which becomes _user's_.
"users" - plural form becomes _users'_

Reply:
Yeah, I missed the `a` and read it as `[...] lets you control the document resources based on the users' attributes [...]`  which feels more natural to me but it's subjective I guess :)